mtga_follower.py

```diff
@@ -137,8 +139,13 @@
 IS_SUCCESS_CODE = lambda code: code >= 200 and code < 300
 DEFAULT_RETRY_SLEEP_TIME = 1
 
+@dataclass
+class MTGAFollowerCliArgs:
+    log_file: str
+    host: str
+    once: bool
```

Comment:
If you can just merge master and add this in, we should be good!

```suggestion
class MTGAFollowerCliArgs:
    log_file: str
    host: str
    once: bool
    token: str
```